File: Basic Coding Problems/01 Fundamentals/002 Sum of Two Numbers/C/SumOfTwoNumbers.c
#include <stdio.h>

int main() {
    int firstNumber, secondNumber, sum;

    printf("Enter the first number: ");
    scanf("%d", &firstNumber);

    printf("Enter the second number: ");
    scanf("%d", &secondNumber);

    sum = firstNumber + secondNumber;

    printf("The sum of the two numbers is: %d\n", sum);

    return 0;
}

File: Basic Coding Problems/01 Fundamentals/002 Sum of Two Numbers/Java/SumOfTwoNumbers.java
import java.util.Scanner;

public class SumOfTwoNumbers {
    public static void main(String[] args) {
        int firstNumber, secondNumber, sum;

        Scanner scanner = new Scanner(System.in);

        System.out.print("Enter the first number: ");
        firstNumber = scanner.nextInt();

        System.out.print("Enter the second number: ");
        secondNumber = scanner.nextInt();

        sum = firstNumber + secondNumber;

        System.out.println("The sum of the two numbers is: " + sum);

        scanner.close();
    }
}